### PR TITLE
Replaces po file parser and fixes problem with personal data in downloaded translations.

### PR DESCRIPTION
**Pull request by ma2ciek**
Fixes https://github.com/ckeditor/ckeditor5-dev/issues/103 and https://github.com/ckeditor/ckeditor5-dev/issues/122.

### CODE REVIEW

**Comment by pomek** [comment]
Missing commit message for merge. See https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention.

File: packages/ckeditor5-dev-utils/tests/translations/cleanpofilecontent.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+/* global describe */
+
+'use strict';
+
+const chai = require( 'chai' );
+const expect = chai.expect;
+const cleanPoFileContent = require( '../../lib/translations/cleanpofilecontent' );
+
+describe( 'translations', () => {
+	describe( 'parsePoFileContent()', () => {
```
Comment (pomek) [inline packages/ckeditor5-dev-utils/tests/translations/cleanpofilecontent.js]: Should it be `it` instead of `describe`, right? :D

Comment (ma2ciek) [inline packages/ckeditor5-dev-utils/tests/translations/cleanpofilecontent.js]: I missed the `it` part :D Thanks.

**Comment by ma2ciek** [comment]
> Missing commit message for merge. See https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention.

Does it apply to the ckeditor5-dev packages as well as to the ckeditor5 packages?

**Comment by Reinmar** [comment]
Yes, we want to start using the same convention in this repo too. But there's no PR template yet.

**Pull request merged**